invertedai/large/initialize.py

```diff
@@ -0,0 +1,265 @@
+import time
+from pydantic import BaseModel, validate_call
+from typing import List, Optional, Tuple
+from itertools import product
+from tqdm.contrib import tenumerate
+import numpy as np
+
+import invertedai as iai
+from invertedai.large.common import Region
+from invertedai.api.initialize import InitializeResponse
+from invertedai.common import TrafficLightStatesDict, Point
+from invertedai.utils import get_default_agent_attributes
+from invertedai.error import InvertedAIError
+
+SLACK = 2
+AGENT_SCOPE_FOV_BUFFER = 20
+
+@validate_call
+def define_regions_grid(
+    map_center: Optional[Tuple[float,float]] = (0.0,0.0),
+    width: Optional[float] = 100.0, 
+    height: Optional[float] = 100.0, 
```

Comment:
Why are these optional? I can't think of a reason you would want to grid an area without knowing its size or position.

Reply:
Good point, made these parameters mandatory.